GitExtensions/Program.cs

```diff
@@ -33,11 +33,8 @@ private static void Main()
             {
                 DiagnosticsClient.Initialize(ThisAssembly.Git.IsDirty);
 
-                if (!Debugger.IsAttached)
-                {
-                    AppDomain.CurrentDomain.UnhandledException += (s, e) => ReportBug((Exception)e.ExceptionObject);
-                    Application.ThreadException += (s, e) => ReportBug(e.Exception);
-                }
+                AppDomain.CurrentDomain.UnhandledException += (s, e) => UncaughtExceptionHandler((Exception)e.ExceptionObject);
+                Application.ThreadException += (s, e) => UncaughtExceptionHandler(e.Exception);
```

Comment:
Could you please explain?
When running under the debugger we want to debugger to handle unhandled exceptions, not have them routed to NBug